changelogs/unreleased/gh-10598-make-connpool-call-faster.md

```diff
@@ -0,0 +1,3 @@
+## feature/connpool
+
+- `connpool.call()` method now works faster (gh-10598).
```

Comment:
I don't like that there are two features in one commit. Furthermore I don't agree that 'works faster' is a feature at all. And as far as I understood the changes, it will only work faster in certain mode, while this changelog isn't clear about that.

I propose we merge this changelog with the other one, just adding there, that this 'alive' mode is faster than the others.

Reply:
Thank you. I've splitted the patch int two distinct commits. Changed the changelog into a bit more specific that the `call()` is now faster only in some specific cases.

Reply:
Seems resolved, so I'm going to close this discussion.